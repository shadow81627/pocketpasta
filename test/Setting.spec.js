import Component from '@/components/settings/setting';
import { shallowMount, createLocalVue } from '@vue/test-utils';
import Vuex from 'vuex';
import Vuetify from 'vuetify';

const localVue = createLocalVue();

localVue.use(Vuex);
localVue.use(Vuetify);

describe('Setting', () => {
  const factory = () => shallowMount(Component, { localVue });

  test('mounts properly', () => {
    const wrapper = factory();
    expect(wrapper.isVueInstance()).toBeTruthy();
  });

  test('renders properly', () => {
    const wrapper = factory();
    expect(wrapper.html()).toMatchSnapshot();
  });
});
