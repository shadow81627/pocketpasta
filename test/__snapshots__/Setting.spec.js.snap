// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Setting renders properly 1`] = `"<li class=\\"list-group-item\\"><div class=\\"row\\"><div class=\\"col-sm-6\\"><v-card-stub loaderheight=\\"4\\" tag=\\"div\\" color=\\"transparent\\" flat=\\"true\\"><v-card-title-stub primarytitle=\\"\\" primary-title=\\"\\"><v-icon-stub left=\\"\\">$</v-icon-stub> <span class=\\"title font-weight-light\\"></span></v-card-title-stub> <v-card-text-stub class=\\"subheading form-text text-muted pt-0\\"></v-card-text-stub></v-card-stub></div> <div class=\\"col-sm-6 align-self-center\\"><input type=\\"text\\" placeholder=\\"placeholder\\" class=\\"form-control\\"></div></div></li>"`;
